app/models/patient.rb

```diff
@@ -122,15 +122,50 @@ def latest_transfer
 
   # Patients who are eligible for reminders
   scope :reminder_eligible, lambda {
```

Comment:
Since this scope has been updated according to PO documentation based on the eligibility flowchart I'm not sure where the optimal eligible fits in anymore. Can it be removed?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/518/files#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR184

Reply:
Yeah I was going to ask for this to be removed as well. This + the redundant checks in `send_assessment`.